OPass/Storyboard/Main.storyboard

```diff
@@ -945,10 +627,30 @@ Please copy and paste the token into next field or click the token link in the r
             </objects>
             <point key="canvasLocation" x="1419" y="1006"/>
         </scene>
+        <!--More-->
+        <scene sceneID="3O0-sW-tMh">
+            <objects>
+                <viewControllerPlaceholder storyboardIdentifier="More" storyboardName="More" id="vOY-Ew-eXR" sceneMemberID="viewController">
+                    <navigationItem key="navigationItem" id="cIe-Uf-tXd"/>
+                </viewControllerPlaceholder>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="Coy-na-vLk" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="2092" y="684"/>
+        </scene>
+        <!--Schedule-->
+        <scene sceneID="q1U-SQ-MXh">
+            <objects>
+                <viewControllerPlaceholder storyboardIdentifier="Schedule" storyboardName="Schedule" id="Qto-S1-Rd1" sceneMemberID="viewController">
+                    <navigationItem key="navigationItem" id="5jP-Yl-PDp"/>
+                </viewControllerPlaceholder>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="hC5-Wc-cQo" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="127" y="684"/>
+        </scene>
         <!--Announce-->
         <scene sceneID="rLM-aC-FYG">
             <objects>
-                <viewController id="ba1-3I-FgV" userLabel="Announce" customClass="AnnounceTableViewController" customModule="OPass" customModuleProvider="target" sceneMemberID="viewController">
+                <viewController id="ba1-3I-FgV" userLabel="Announce" customClass="AnnounceTableViewController" sceneMemberID="viewController">
```

Comment:
since `AnnounceTableViewController` was converted to Swift, remove `customModule="OPass" customModuleProvider="target"` won't execute correctly, it always fails and crash when loaded